etna/models/__init__.py

```diff
@@ -8,6 +8,8 @@
 from etna.models.catboost import CatBoostModelPerSegment
 from etna.models.catboost import CatBoostMultiSegmentModel
 from etna.models.catboost import CatBoostPerSegmentModel
+from etna.models.deadline_ma import DeadlineMovingAverageModel
+from etna.models.deadline_ma import _DeadlineMovingAverageModel
```

Comment:
actually you don't need to import "private" class `_DeadlineMovingAverageModel` here, only `DeadlineMovingAverageModel`